scapy/fields.py

```diff
@@ -3520,7 +3520,12 @@ def i2repr(self, pkt, x):
         elif self.custom_scaling:
             x = x / self.custom_scaling
         x += self.delta
-        t = datetime.fromtimestamp(x).strftime(self.strf)
+        # To make negative timestamps work on all plateforms (e.g. Windows),
+        # we need a trick.
+        t = (
+            datetime.datetime(1970, 1, 1) +
+            datetime.timedelta(seconds=x)
+        ).strftime(self.strf)
```

Comment:
Looks like it can overflow:
```python
  File "/home/vagrant/scapy/scapy/fields.py", line 225, in i2repr
    return repr(self.i2h(pkt, x))
           ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/vagrant/scapy/scapy/packet.py", line 572, in __repr__
    val = f.i2repr(self, fval)
          ^^^^^^^^^^^^^^^^^^^^
  File "/home/vagrant/scapy/scapy/layers/ntlm.py", line 157, in i2repr
    return repr(self._on_payload(pkt, x, "i2repr"))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/vagrant/scapy/scapy/packet.py", line 572, in __repr__
    val = f.i2repr(self, fval)
          ^^^^^^^^^^^^^^^^^^^^
  File "/home/vagrant/scapy/scapy/fields.py", line 3526, in i2repr
    datetime.datetime(1970, 1, 1) +
OverflowError: date value out of range
```
```python
repr(Loopback(b"\n\x00\x00\x87\x87\x87\x87\x87\x88\x06\x06\x06\x06\x06\x06\xc7\x06\x06\x06\x06`\x06\x06\x06\x06\x06\x06\x06\x06\x06\x06\x06\x06\x00\x08\x00\x00\x00\x00\x00\x00\x00\x00\x00\n\x00\x00\x87\xff\xff\xff\xff\xff\xff3?\x00\x00\x00\r\r\x18\x18\x18\x18\r\r\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\x03\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\xff\x87\xd4\xc3\xb2\xa1\x00\x00\x00\x00\x00\xff1\x00\x02\x00\x00\x06\x00\x00\n\x00\x00\x80x\x87\x87\x87\x88\x06\x06\x06\x06\x06\x06\xc7\x06\x06\x06\x06`\x06\x06\x06\x06\x06\x06\x04\x06\x06\x06\x06\x06\x00\x08\x00\x00\x00\x00\x00\x00\x00\x00\x00\n\x00\x00\x87\xff\xff\xff\xff\xff\xff3?\x00\x00\x00\r\r\x18\x18\x18\x18\r\r\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\xff\x03\xf8\x00\x00\x00\x00\x00\x00\x00\x00\x00\xff\x87\xd4\xc3\xb2\xa1\x00\x00\x00\x00\x00\x00\x00\x00\x00\x06\x06\x06\x00\x08\x00\x00\x00\x00\x00\x00\x00\x00\x00\n\x00\x00\x87\xff\xff\xff\xff\xff\xff3?\x00\x00\x00\x00\x01\xb9\x00\xf9\xff\xff\xff\xff\xff\xff\xff\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00M'\x00\x00\x00\x00\x00\x00\x01M \x01\x00\x00\x00"))
```
My knee-jerk reaction was to catch the OverflowError exception and assign something like `<invalid>` to `t` but I'm not sure if it's the right fix.